### DISCUSSION TITLE
local development, are there any way to be able to send post request to another http server?

### DISCUSSION THREAD

**Discussion by Katsukiniwa** [nuxt/nuxt#17963] [Questions] [answered]
I develop browser application with nuxt3 and server side application which is RESTful API server is Ruby on Rails.
now, I implement simple post request from nuxt3 to rails like this.
Rails application is work on port 3000, and nuxt3 is on 8000.

```ts
  const { data, pending } = useFetch('/articles/create', {
    method: 'POST',
    params: { title },
    headers: {
      "Accept": "application/json",
      "Content-Type": "application/json",
    },
    baseURL: 'http:/localhost:3000'
  })
```

But, this code create post request to `http://127.0.0.1:8000/localhost:3000/articles/create?title=foo`.
Therefor, change `baseURL` to `https://localhost:3000`, nuxt3 request to `https://localhost:3000/articles/create?title=foo`.
On local development, I don't want to Self-signed certificate of SSL because it's troublesome.
So, I want to ask how to create post request to with `http`.

**Comment by cperrin88** [comment]
What happens if you just do `useFetch('http://localhost:3000/articles/create/articles/create', ...)`?

**Reply by Katsukiniwa** [reply]
> fetch.mjs?v=2209c45f:131 POST http://localhost:8000/localhost:3000/articles/create?title=foo 404 (Internal Server Error)

happen.

**Comment by wobsoriano** [comment, marked as answer]
You have a typo in your baseUrl.

`http:/localhost:3000` is missing one `/`

**Reply by Katsukiniwa** [reply]
Oh, that's right....
Sorry...
Thank your advice!
